### ISSUE TITLE
acl.role_ttl: invalid duration: "20": time: missing unit in duration 20

### ISSUE THREAD

**Issue by ghost** [hashicorp/consul#5905]
Please search the existing issues for relevant feature requests, and use the reaction feature (https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to add upvotes to pre-existing requests.

#### Feature Description

When using this config:
```
acl = {
  role_ttl                  = "20",              # 1.4.4 default 30
}
```
I get:

# /usr/local/bin/consul agent -config-dir=/etc/consul.d/
```
==> 1 error occurred:
	* acl.role_ttl: invalid duration: "20": time: missing unit in duration 20
```

and I wondered if this [could be documented](https://www.consul.io/docs/agent/options.html#acl_role_ttl) please - it would be good to know the required units before I configure the servers.  I can't find anything specific about HCL1 time specifications either.

#### Use Case(s)

As a Consul user, I would like to know any required units used in the Consul config by reading the Consul config docs.

**Comment by mkeeler** [comment]
We will need to document this. For now you can use things like "20s" "5m" "1h" or anything that is supported by the golang [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration) function:

> ParseDuration parses a duration string. A duration string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".

**Comment by blake** [comment]
This should be resolved by #6693.
